snippets/Tooltip.md

```diff
@@ -46,25 +55,29 @@ class Tooltip extends React.Component {
   };
 
   render() {
-    const { children, ...rest } = this.props;
+    const { children, text, ...rest } = this.props;
     const { show } = this.state;
-    const { visible, tooltip } = this.style;
+    const { visible, tooltip, tooltipArrow } = this.style;
     const showTooltip = show ? visible : {};
     return (
       <div>
         <div style={{ ...tooltip, ...showTooltip }}>
-          This is a simple tooltip
+          {text}
+          <span style={tooltipArrow}/>
+        </div>
+        <div {...rest} onMouseEnter={this.showTooltip} onMouseLeave={this.hideTooltip}>
+          {children}
```

Comment:
@skatcat31 Now children are wrapped in a div and rendered instead of being cloned with new props.